core/client/app/templates/components/gh-profile-image.hbs

```diff
@@ -1,11 +1,11 @@
 <figure class="account-image js-file-upload">
     {{#unless hasUploadedImage}}
-        <div class="placeholder-img" style={{defaultImage}}></div>
-
         {{#if hasEmail}}
```

Comment:
Gravatar default images are horrible (and defeat the purpose of this feature) - we explicitly specify *no* default should be returned in our request to the Gravatar API for this reason.